components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -1799,4 +1799,20 @@ public static String getInitiatorId(String userName, String tenantDomain) {
         }
         return userId;
     }
+
+    /**
+     * Read the SCIM User Endpoint Maximum Items Per Page is enabled config and returns it.
+     *
+     * @return If SCIM User Endpoint Maximum Items Per Page is enabled.
+     */
+    public static boolean isSCIM2UserMaxItemsPerPageEnabled() {
+
+        String scim2UserMaxItemsPerPageEnabledProperty =
+                IdentityUtil.getProperty(IdentityCoreConstants.SCIM2_USER_MAX_ITEMS_PER_PAGE_ENABLED);
+
+        if (StringUtils.isBlank(scim2UserMaxItemsPerPageEnabledProperty)) {
```

Comment:
Here if the configuration value is deliberately set to an empty string (or just spaces), based on the original logic, it would imply that the SCIM User Endpoint Maximum Items Per Page feature is enabled, which does not align with the intended use of the configuration. Hence remove if condition :
`if (StringUtils.isBlank(scim2UserMaxItemsPerPageEnabledProperty)) `